src/components/Theme/src/default-theme.js

```diff
@@ -16,6 +16,10 @@ export default function defaultTheme() {
 			overlay: 'rgba(0, 0, 0, 0.3)',
 			primary: '#000000',
 		},
+		fonts: {
+			baseSize: '14px',
+			sizeScale: 1.15,
```

Comment:
Changing the base font size and scale is going to have negative consequences on sites that are currently using 16px as the base font size. We should not change these defaults at this time.